src/styles/bloks/card.scss

```
@@ -0,0 +1,100 @@
+.card {
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  padding: 32px 16px 16px;
+  transition: transform 0.3s;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__icon {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+    display: block;
+  }
+
+  &__info {
+    text-align: left;
+    color: #060b35;
+    margin-bottom: 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__info:hover {
+    color: #34568b;
+  }
+
+  &__code {
+    margin: 0 0 7px;
+    color: #616070;
+
+    font-size: 10px;
+
+    line-height: 14px;
+  }
+
+  &__rate {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 15px;
+  }
+
+  &__rate-text {
+    color: #060b35;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 1.4;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    font-size: 12px;
+    margin-bottom: 15px;
+  }
+
+  &__price-text {
+    color: #616070;
+    line-height: 18px;
+    font-size: 12px;
+    font-weight: 400;
+    margin: 0;
+  }
+
+  &__price-money {
+    color: #060b35;
+    margin: 0;
+    font-size: 16px;
+    font-weight: 700;
+  }
```

Use scss nesting, fix it everywhere 
```suggestion
  &__price {
    display: flex;
    justify-content: space-between;
    align-items: center;
    font-size: 12px;
    margin-bottom: 15px;
    
    &-text {
      // write styles here
    }
    
    &-money {
      // write styles here
    }
  }
```